event_seats_per_registration/__openerp__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U.
+
+{
+    "name": "Seats per registration in events",
+    "version": "8.0.3.1.0",
```

Comment:
It's because this module was already in another repository before https://github.com/OCA/event existed, and it had versions back there. If you don't care much, I'd prefer to leave it as is.